### DISCUSSION TITLE
Web app to Publii site which lets technical person easily add and remove notice text

### DISCUSSION THREAD

**Discussion by konstantin-fi** [GetPublii/Publii#1493] [Help] [answered]
Hello!

My customer has a static html css Publii site and would like to be able to add a basic notification text box to the site with an expiration date. Management must be as simple as entering a login and password, receiving a graphical user interface with a topic, text, expiration date, and SAVE, and then logging out. Is this achievable with Publii, or do I need another integration with the site, such as a skeleton WordPress window or another comparable web app? Could you perhaps recommend something if Publii is unable to create such an app?

**Comment by dziudek** [comment, marked as answer]
Hi,

For such feature please consider some 3rd party service which allows website owners to display some notification bars etc. Something like convertbox.com

**Comment by konstantin-fi** [comment]
I thank you.
